jvm-packages/xgboost4j-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostConfigureSuite.scala

```diff
@@ -77,4 +80,27 @@ class XGBoostConfigureSuite extends FunSuite with PerTest {
     }
     ss.conf.unset("xgboost.spark.ignoreSsl")
   }
+
+  test("Check setting rabit configs no affect on prediction") {
```

Comment:
we should also have set of tests in xgb-spark ensuring that, changing rabit params can still make training (approx&hist) run with verified results

Reply:
for this particular test, it is confusing to me that why "prediction" is something error-prone when there is changing of rabit params?

Reply:
good call, In fact, we should only check if parameters were propagated into rabit. As stated above, this pr doesn't suppose to change behavior of xgb-spark.

Reply:
moved to rabitRegressionSuite